mechanics.md

```diff
@@ -4,63 +4,65 @@ So, how do I use this?
 
 ## I Have an Idea!
 
-Ideas should have
- * a pretty specific goal: a user story ("Users can..."), dependency ("This will allow ..") or a completion condition ("All services .."); and
- * some vague idea of how this might be accomplished, or a few alternatives.
+We've learned that ideas should be developed outside of the RFC process first.
+Create a document in another context and talk it through with the relevant people.
+Keep iterating until there is one option that has rough agreement from those people.
+It's OK if some details are still unclear.
 
-If you have an idea, here's how to get started:
+A symptom of rushing this phase is an enormous list of comments on the RFC pull request.
+This is usually difficult to follow and communications issues quickly swamp the process.
+If this occurs, return to the idea phase.
+
+## I Think We Have Rough Consensus!
```

Comment:
I wonder if RFCs should contain the alternatives considered and which trade-offs were chosen in the end?